packages/client/tests/functional/composites-mongo-list/_matrix.ts

```diff
@@ -0,0 +1,21 @@
+import { defineMatrix } from '../_utils/defineMatrix'
```

Comment:
I think we can rename the folders to `composites-list` and `composites-object` as composites might come for more providers in the future.